src/credentials/CHIPCertFromX509.cpp

```diff
@@ -693,5 +694,40 @@ DLL_EXPORT CHIP_ERROR ConvertX509CertToChipCert(const uint8_t * x509Cert, uint32
     return err;
 }
 
+CHIP_ERROR ConvertX509CertsToChipCertArray(const ByteSpan & x509Cert1, const ByteSpan & x509Cert2, uint8_t * chipCertArrayBuf,
```

Comment:
Suggest having an arg that is ICAC and another that is NOC, so that more validity checks can be provided (e.g. NOC always present).

The validity check that NOC's issuer matches the ICAC, if present, should be enforced if possible.